native/src/routes/jpal-tracking/JpalTracking.js

```diff
@@ -0,0 +1,141 @@
+// @flow
+
+import React, { useEffect, useMemo, useState } from 'react'
+
+import type { ThemeType } from 'build-configs/ThemeType'
+import { Switch, Text, TextInput, View } from 'react-native'
+import { useTranslation } from 'react-i18next'
+import type { StateType } from '../../modules/app/StateType'
+import AppSettings, { defaultSettings } from '../../modules/settings/AppSettings'
+import type { NavigationPropType, RoutePropType } from '../../modules/app/constants/NavigationTypes'
+import type { JpalTrackingRouteType } from 'api-client'
+import styled from 'styled-components/native'
+import LayoutContainer from '../../modules/layout/containers/LayoutContainer'
+import withTheme from '../../modules/theme/hocs/withTheme'
+import Caption from '../../modules/common/components/Caption'
+
+const ThemedText = styled.Text`
+  display: flex;
+  text-align: left;
+  color: ${props => props.theme.colors.textColor};
+  font-family: ${props => props.theme.fonts.decorativeFontRegular};
```

Comment:
Otherwise the app crashes:
```suggestion
  font-family: ${props => props.theme.fonts.native.decorativeFontRegular};
```